QUESTIONS.md

```diff
@@ -3,3 +3,8 @@
 What questions can you come up with?
 
 ## Cohort Questions
+question1:
```

Comment:
Great question! I wonder if we would benefit from gaining more education experience.